src/main/java/picocli/CommandLine.java

```diff
@@ -13483,11 +13483,45 @@ public void addEmptyRow() {
             /** Delegates to {@link #addRowValues(CommandLine.Help.Ansi.Text...)}.
              * @param values the text values to display in each column of the current row */
             public void addRowValues(String... values) {
-                Text[] array = new Text[values.length];
-                for (int i = 0; i < array.length; i++) {
-                    array[i] = values[i] == null ? Ansi.EMPTY_TEXT : ansi.new Text(values[i]);
+                Text[] rowValues = new Text[values.length];
+                for (int i = 0; i < rowValues.length; i++) {
+                    rowValues[i] = values[i] == null ? Ansi.EMPTY_TEXT : ansi.new Text(values[i]);
+                }
+                addRowValues(rowValues);
+            }
+            /** Delegates to {@link #addRowValues(CommandLine.Help.Ansi.Text...)}.
+             * @param values the text values to display in each column of the current row, splitting for newlines*/
+            public void addRowValuesWithNewlines(String... values) {
```

Comment:
Shall we simply replace the existing `addRowValues(String...)` method with the new implementation, instead of adding a new `addRowValuesWithNewlines` method? Chances are that most callers prefer to use the new logic anyway.

Reply:
Sure.